### PR TITLE
Chore: e2e tests on gmcq component (Issue/177)

### PR DESCRIPTION
**Pull request by lemmyadams**
Addresses #177

### CODE REVIEW

**Comment by lemmyadams** [comment]
@cahirodoherty-learningpool on this test and the other questions I test for the buttons appearing as expected, for the submit and show feedback. 
The code is:
function testQuestionButtons(buttonsObject = {btn__action: 'Submit', btn__feedback: 'Show feedback'}) {
  Object.keys(buttonsObject).forEach((key) => {
    cy.get(`.${key}`).should('contain', buttonsObject[key])
  })
}
Just getting your thoughts before I raise an issue on the framework

**Comment by cahirodoherty-learningpool** [comment]
> @cahirodoherty-learningpool on this test and the other questions I test for the buttons appearing as expected, for the submit and show feedback. The code is:
> 
> ```
> function testQuestionButtons(buttonsObject = {btn__action: 'Submit', btn__feedback: 'Show feedback'}) {
>   Object.keys(buttonsObject).forEach((key) => {
>     cy.get('.${key}').should('contain', buttonsObject[key])
>   })
> }
> ```
> 
> Just getting your thoughts before I raise an issue on the framework

This looks okay on first inspection anyway. As long as the buttonsObject is passed as a param for each plugin that uses this helper 👍

**Comment by lemmyadams** [comment]
Relies on https://github.com/adaptlearning/adapt_framework/pull/3533

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]

File: test/e2e/gmcq.cy.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-        cy.testContainsOrNotExists('.gmcq-item__text', item.text)
-      })
+      const stripHtml = cy.helpers.stripHtml;
+      cy.testContainsOrNotExists('.gmcq__body', stripHtml(gmcqComponent.body));
+      cy.testContainsOrNotExists('.gmcq__title', stripHtml(gmcqComponent.displayTitle));
+      cy.testContainsOrNotExists('.gmcq__instruction', stripHtml(gmcqComponent.instruction));
+
+      cy.get('.gmcq-item').should('have.length', gmcqComponent._items.length);
+      gmcqComponent._items.forEach(item => {
+        cy.testContainsOrNotExists('.gmcq-item__text', item.text);
```
Comment (oliverfoster) [inline test/e2e/gmcq.cy.js:18]: Please check all of the tested properties for the `compile` helper. That's where the json data is processed as a handlebars template and can produce html.

https://github.com/adaptlearning/adapt-contrib-gmcq/blob/eeadd8f17e1752763a62f3b014d5234380e6b1d0/templates/gmcq.jsx#L127

**Review by oliverfoster — CHANGES_REQUESTED** [review]

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**